deployer/deploy.sh

```diff
@@ -23,8 +23,12 @@ osds=$((${QUOTA_COUNT:-1} * 3))
 echo "Quota count: ${QUOTA_COUNT} -- OSD count: ${osds}"
 
 #-- Patch the OSD count into the StorageCluster & apply it
+#-- If config map containing StorageCluster exists, apply it. 
+#-- Else, apply the default StorageCluster.
 while true; do
-    sed "s/STORAGE_NODES/${osds}/" storagecluster.yml | \
+    storageClusterPath="storagecluster.yml"
+    [ -f /sc-override/storagecluster.yml ] && storageClusterPath="/sc-override/storagecluster.yml"
```

Comment:
Can we move `/sc-override/storagecluster.yml` into an env var? it is used more than once

Reply:
fixed!